docs/examples.md

```diff
@@ -86,17 +87,17 @@ libraryDependencies ++= List(
 
 Example code:
 
-```scala
+```scala mdoc:compile-only
 import sttp.client._
 import sttp.client.circe._
 import sttp.client.asynchttpclient.zio._
 import io.circe.generic.auto._
-import zio._
+import zio.{App => ZApp, _}
 import zio.console.Console
 
-object GetAndParseJsonZioCirce extends App {
+object GetAndParseJsonZioCirce extends ZApp {
```

Comment:
I'd use `zio.App` directly

Reply:
These files are copy-pasted from files in the `examples` project. Maybe we should drop them and just leave links, or embed?

Reply:
What do you mean by embedding? I am not aware of such feature in mdoc